src/sentry/integrations/client.py

```diff
@@ -160,6 +160,8 @@ def _request(self, method, path, headers=None, data=None, params=None,
         full_url = self.build_url(path)
         host = urlparse(full_url).netloc
         session = build_session()
+
+        metrics.incr('integrations.http_request', tags={'host': host})
```

Comment:
Does this have the ability to turn into an IP for things like webhooks? I can see that being a fairly potential downside here.

Reply:
It could depending on the integration type. I can update the various integration clients to pass in a more fixed named. We'll lose some visibility into customer specific issues but have fewer tags in datadog.

Reply:
@JTCunning Updated to use the integration name instead of host as a tag. This will result in only a handful of tags instead of a theoretically unbounded number.